hw-tictac/src/components/header/HeaderMobile.jsx

```diff
@@ -22,116 +13,106 @@ const WALLET_MAC_URL = process.env.REACT_APP_WALLET_MAC_URL;
-            location: EXPLORER_URL,
-            displayCallback: ({ location, label }) =>
-                <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
-        },
-        {
-            label: "About",
-            location: ALICENET_URL,
-            displayCallback: ({ location, label }) =>
-                <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
-        },
-        {
-            label: "GitHub",
-            location: GITHUB_URL,
-            displayCallback: ({ location, label }) =>
-                <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
-        },
-        {
-            label: "White Paper",
-            location: WHITE_PAPER_URL,
-            displayCallback: ({ location, label }) =>
-                <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
-        },
-    ];
+const sections = [
+    {
+        label: "Block Explorer",
+        location: EXPLORER_URL,
+        displayCallback: ({ location, label }) => (
+            <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
+        ),
+    },
+    {
+        label: "About",
+        location: ALICENET_URL,
+        displayCallback: ({ location, label }) => (
+            <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
+        ),
+    },
+    {
+        label: "GitHub",
+        location: GITHUB_URL,
+        displayCallback: ({ location, label }) => (
+            <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
+        ),
+    },
+    {
+        label: "White Paper",
+        location: WHITE_PAPER_URL,
+        displayCallback: ({ location, label }) => (
+            <MenuItem key={`header-mobile-${label}`} location={location} label={label} blank />
+        ),
+    },
+];
 
-const subSections =
-    [
-        {
-            label: "iOS",
-            location: WALLET_MAC_URL,
-            icon: <FontAwesomeIcon icon={faApple} />,
-            displayCallback: ({ location, label, icon }) =>
-                <MenuItem key={`wallet-section-${label}`} icon={icon} location={location} label={label} blank />
-        },
-        {
-            label: "Linux",
-            location: WALLET_LINUX_URL,
-            icon: <FontAwesomeIcon icon={faLinux} />,
-            displayCallback: ({ location, label, icon }) =>
-                <MenuItem key={`wallet-section-${label}`} icon={icon} location={location} label={label} blank />
-        },
-        {
-            label: "Windows",
-            location: WALLET_WINDOWS_URL,
-            icon: <FontAwesomeIcon icon={faWindows} />,
-            displayCallback: ({ location, label, icon }) =>
-                <MenuItem key={`wallet-section-${label}`} icon={icon} location={location} label={label} blank />
-        },
-    ];
+// const subSections =
```

Comment:
Considering we are using a versioning system, I would consider removing the code instead. It's less confusing at least to me.

Reply:
Sounds good to me , wasn't sure if you needed it -- Will remove